modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogTableDescriptor.java

```diff
@@ -96,6 +101,10 @@ public int zoneId() {
         return zoneId;
     }
 
+    public int primaryKeyId() {
+        return primaryKeyId;
```

Comment:
* Personally I prefer `pkIndexId` for the variable name and `primaryKeyIndexId()` for this method 
* It looks strange that we have pkColumns and pkIndexId at the same time, as far as I understand, we can always check that it is a PK index if it consists of the 'pkColumns', right? ...Or vice-versa, we should be able to get a list of pk columns by pkIndexId.

Reply:
renamed